Automattic/pocket-casts-ios
# Fix auto play when skipping last

| 📘 Part of: # |  
|:---:|

Fixes #2019 



Updates the code to make sure that auto-play logic and other end of episode logic is activated when skip last is triggered.

## To test

- Start the app
- Ensure that you have Auto-Play active ( Profile -> Settings -> General -> Autoplay )
- Clean your Up Next Queue
- Subscribe to a podcast
- On the podcast page tap on the cog icon on the top right
- Setup a Skip Last time. Ex: 15 seconds
- Ensure that podcast has multiple episodes available
- Play an episode of the podcast
- Skip forward near to the skip last interval
- await for the skip last to kick in
- see if the auto-play works and a new episode of the same podcast is added to your queue and starts playing.

## Checklist
